src/include/ceph_fs.h

```diff
@@ -231,6 +231,7 @@ struct ceph_mon_subscribe_ack {
  */
 #define CEPH_MDSMAP_DOWN    (1<<0)  /* cluster deliberately down */
 #define CEPH_MDSMAP_ALLOW_SNAPS   (1<<1)  /* cluster allowed to create snapshots */
+#define CEPH_FUSE_NO_MDS_UP    -(1<<2) /* no mds up deteced in ceph_fuse */
```

Comment:
I like your instinct to create a defined constant for this state, but include/ceph_fs is not the right place for a code-local interface like that — as it says at the top of the header
> ceph_fs.h - Ceph constants and data types to share between kernel and user space. Most types in this file are defined as little-endian, and are primarily intended to describe data structures that pass over the wire or that are stored on disk.

I think for now putting it in Client.h is probably fine, and if we want to export it to libcephfs users then it can re-export to them as well.